projects/media-viewer/src/lib/toolbar/main-toolbar/main-toolbar.component.html

```diff
@@ -1,22 +1,14 @@
-<mv-search-bar></mv-search-bar>
 <mv-sub-toolbar></mv-sub-toolbar>
 <div class="toolbar">
   <div id="toolbarContainer">
-    <div id="toolbarViewer">
-      <mv-tb-left-pane></mv-tb-left-pane>
-      <mv-tb-middle-pane></mv-tb-middle-pane>
-      <mv-tb-right-pane
-        [enableRedaction]="enableRedactions"
-        [enableICP]="enableICP"
-        [enableAnnotations]="enableAnnotations"
-      >
-      </mv-tb-right-pane>
-    </div>
+    <mv-media-viewer-toolbar [enableRedaction]="enableRedactions" [enableICP]="enableICP"
```

Comment:
I think we can combine the main-toolbar and the media-viewer-toolbar, otherwise it's not clear what does what from the names

Reply:
Agreed. This will be resolved in the next commit.